test/hosting/hosting.tests.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="../Test.Common.props"/>
+
+  <PropertyGroup>
+    <Description>PowerShell hosting SDK xUnit Tests</Description>
+    <AssemblyName>powershell-hosting-tests</AssemblyName>
+    <RuntimeIdentifiers>win7-x86;win7-x64;osx.10.12-x64;linux-x64</RuntimeIdentifiers>
```

Comment:
Just a question, is the `RuntimeIdentifiers` tag needed? It seems we are building an assembly instead of an app, so it shouldn't be necessary.

Reply:
Removed it.